scripts/globals.d.ts

```diff
@@ -37,8 +44,14 @@ interface Window {
     webkitURL: any;
     superdeskConfig: any;
     module: any;
+    RunTansaProofing: any;
+    iframely: any;
 }
 
+// Allow importing json/html files
+declare module "*.json";
```

Comment:
Where is that used? I don't think importing json/html was supported.

Reply:
For example here https://github.com/superdesk/superdesk-client-core/blob/master/scripts/core/editor3/html/tests/from-html-snapshot.spec.jsx#L1-L2

Reply:
Yeah, I added that test, but it's been disabled for ages. Importing json/html is pretty handy for snapshot testing, so let's keep it.